api/Cargo.toml

```diff
@@ -18,8 +18,8 @@ serde = "1.0.136"
 structopt = "0.3.26"
 sqlx = { version = "0.5", features = ["runtime-tokio-native-tls", "postgres"] }
 tokio = { version = "1.15.0", features = ["full"] }
-# not great, but the changes for hyper 0.14 and tokio 1 have not been merged on the main repo
-hyper-reverse-proxy = { git = "https://github.com/jtroo/hyper-reverse-proxy" }
+# not great, but waiting for WebSocket changes to be merged
+hyper-reverse-proxy = { git = "https://github.com/chesedo/hyper-reverse-proxy", branch = "master" }
```

Comment:
Seems this was [merged upstream](https://github.com/felipenoris/hyper-reverse-proxy/pull/35).

Reply:
Yip, it's just [not released yet](https://github.com/felipenoris/hyper-reverse-proxy/releases) :disappointed: